### DISCUSSION TITLE
Panel Component v9

### DISCUSSION THREAD

**Discussion by LoganBwDE** [microsoft/fluentui#26009] [General] [answered]
Hi,

is there a Panel Component in v9 or is it planned to move it also to v9 in the future? Or is the Panel obsolete and you should create it on your own? :)

**Comment by mimashay** [comment]
Bumping this! We need one too and I'm this close | | to just rolling my own.  @LoganBwDE did you end up building yours?

**Reply by LoganBwDE** [reply]
@mimashay  Nope, just didnt move the component from v8 to v9 yet ^^

**Comment by khmakoto** [comment, marked as answer]
Hi everyone! We are planning to bring the `Panel` component to v9 in the future.

CC'ing @JustSlone and @paulgildea for more details on timing.

**Reply by mimashay** [reply]
What is the recommendation to use/do in the meantime?

**Reply by nandox5** [reply]
Stick to v8 till all your needed components are migrated. Or create your own component, there are lots of examples out there for creating drawers/panels

**Reply by blendsdk** [reply]
Any news on this?

**Reply by mimashay** [reply]
FWIW we ended up sticking with v9.  I built a Panel component for our project to fill in the gap (let me know if anyone wants to borrow it!...its not perfect...yet but usable).

**Reply by khmakoto** [reply]
We are starting work on a v9 `Drawer` component (the successor of v8's `Panel`). Feel free to follow it [in its epic](https://github.com/microsoft/fluentui/issues/26420).

**Reply by shree998** [reply]
> Stick to v8 till all your needed components are migrated. Or create your own component, there are lots of examples out there for creating drawers/panels

@nandox5 Could you please provide some examples or documentation to create custom panels in V9?

**Reply by shree998** [reply]
> FWIW we ended up sticking with v9. I built a Panel component for our project to fill in the gap (let me know if anyone wants to borrow it!...its not perfect...yet but usable).

@mimashay Please provide me the panel.

**Reply by marcosmoura** [reply]
The Drawer component has been available for a while now. 
You can check the examples and how to use it here:
https://react.fluentui.dev/?path=/docs/components-drawer--default
